docs/docs/articles/github-actions.md

```diff
@@ -98,38 +89,28 @@ jobs:
       # Get the GKE credentials so we can deploy to the cluster
       - uses: google-github-actions/get-gke-credentials@fb08709ba27618c31c09e014e1d8364b02e5042e
         with:
-          cluster_name: ${{ env.GKE_CLUSTER_NAME_DEV }}
-          location: ${{ env.GKE_ZONE_DEV }}
+          cluster_name: ${{ env.GKE_CLUSTER_NAME }}
+          location: ${{ env.GKE_ZONE }}
           credentials: ${{ secrets.GKE_SA_KEY }}
 
-      # Run Testkube test on a GKE cluster
-      - name: Run test
-        id: run_test
-        uses: kubeshop/testkube-docker-action@v1
+      - name: Install Helm
+        uses: azure/setup-helm@v3
         with:
-          command: run
-          resource: test
-          parameters: TEST_NAME
-```
-
-Along with the `kubectl` command, you can pass all the standard K8s parameters such as `--namespace`, etc.
-
-If you wish to automate the CI/CD part of Testkube's Helm release, use `Helm` blocks as follow:
-
-```yaml
-# ...
-- name: Install Helm
-  uses: azure/setup-helm@v1
-  with:
-    version: v3.4.0
-
-- name: Installing repositories
-  run: |
-    helm repo add kubeshop https://kubeshop.github.io/helm-charts
-    helm repo add bitnami https://charts.bitnami.com/bitnami
-
-# Run Helm delpoy/upgrade of the Testkube release on a GKE cluster
-- name: Deploy
-  run: |-
-    helm upgrade --install --atomic --timeout 180s testkube kubeshop/testkube --namespace testkube --create-namespace
+          version: v3.10.0
+          
+      - name: Installing repositories
+        run: |
+          helm repo add helm-charts https://kubeshop.github.io/helm-charts
+          helm repo add bitnami bitnami https://charts.bitnami.com/bitnami
+
+      # Deploy Testkube helm chart on a GKE cluster
+      - name: Deploy
+        run: |-
+          helm upgrade --install --atomic --timeout 180s ${{ env.DEPLOYMENT_NAME }} helm-charts/testkube --namespace testkube --create-namespace
+          
+    # Run a test
+      - uses: kubeshop/setup-testkube@v1
+      - run: |
+          testkube create test --name some-test-name --file path_to_file.json
+          testkube run test some-test-name
```

Comment:
Shouldn't it be still split? I would not encourage Users to install Testkube each time.

Reply:
what do you mean by 'split'?  We deploy TK to the cluster (helm) and install a client using  `kubeshop/setup-testkube@v1` and that's it, it is installed once.

Reply:
Earlier, the deployment was in a separate snippet, split with

> Along with the `kubectl` command, you can pass all the standard K8s parameters such as `--namespace`, etc.
>
> If you wish to automate the CI/CD part of Testkube's Helm release, use `Helm` blocks as follow:

In the new documentation, I feel like the User may think that it's expected to install the Testkube each time before using `kubeshop/setup-testkube`. I'm not 100% sure though

Reply:
ah, understood. This is the complete example of helm+ testkube cli, similar to what we have in our flow so I wouldn't remove it. And above examples explain well what is done. let's ask @TheBrunoLopes